src/webtoon_downloader.py

```diff
@@ -313,15 +319,28 @@ def download_chapter(chapter_download_task_id: int, session: requests.Session, v
         os.makedirs(dest)
     progress.update(chapter_download_task_id, total=len(img_urls), rendered_total=len(img_urls))
     progress.start_task(chapter_download_task_id)
+    image_download_futures = set()
     with ThreadPoolExecutorWithQueueSizeLimit(maxsize=10, max_workers=4) as pool:
         for page_number, url in enumerate(img_urls):
-            pool.submit(download_image, chapter_download_task_id, url, dest, chapter_info.chapter_number, page_number, image_format=images_format)
+            image_download_futures.add(
+                pool.submit(download_image, chapter_download_task_id, url, dest, chapter_info.chapter_number, page_number, image_format=images_format)
```

Comment:
Images will still be downloaded to the provided `dest`, so ending up with a  copy of the images and the .cbz as well

Reply:
This function needs to change anyways in order to allow for .cbz conversion in memory maybe.

Reply:
Yeah, I left the existing files and folder in place because I didn't want to assume too much about what should change. I mostly added this to scratch my own itch of not wanting to deal with 150+ folders to compress for the toon I wanted to download.